analyze/analyses/paths/mod.rs

```diff
@@ -0,0 +1,145 @@
+use std::collections::BTreeSet;
+
+use regex;
+
+use twiggy_ir as ir;
+use twiggy_opt as opt;
+use twiggy_traits as traits;
+
+mod paths_emit;
+mod paths_entry;
+
+use self::paths_entry::PathsEntry;
+
+#[derive(Debug)]
+struct Paths {
+    items: Vec<ir::Id>,
+    opts: opt::Paths,
+    entries: Vec<PathsEntry>,
```

Comment:
Whoops, I need to clean up the previous `items` member here, I'll take care of that.